CHANGELOG.md

```diff
@@ -1,5 +1,33 @@
 # Changelog
 
+## [v0.18.1](https://github.com/operator-framework/operator-lifecycle-manager/tree/v0.18.1) (2021-05-05)
+
+[Full Changelog](https://github.com/operator-framework/operator-lifecycle-manager/compare/v0.18.1...v0.18.1)
+
+**Other changes:**
+
+- Fix mishandling of log format strings in Kind test provisioner. [\#2141](https://github.com/operator-framework/operator-lifecycle-manager/pull/2141) ([benluddy](https://github.com/benluddy))
```

Comment:
I stole in this PR before you generated the changelog.

Reply:
you're suggesting that we remove this commit from the changelog manually? that sounds fine